src/admin/pages/Settings.vue

```diff
@@ -405,6 +406,28 @@
 
                 self.settingFields = settingFields;
                 self.settingSections = settingSections;
+            },
+
+            handleDataClearCheckboxEvent() {
+                let self = this;
+                $('#dokan_general\\[data_clear_on_uninstall\\]').on('click', function () {
+                    if( $(this).is(':checked') ) {
+                        self.$swal({
+                            title: self.__( 'Are you sure?', 'dokan-lite' ),
+                            type: 'warning',
+                            html: self.__( 'All the data will be clear after uninstall and you will not able to recover that.', 'dokan-lite' ),
```

Comment:
kindly change this error message into: All data and tables related to Dokan and Dokan Pro will be deleted permanently after deleting the Dokan plugin. You will not be able to recover your lost data unless you keep a backup. Are you sure you want to enable this feature?